components/EditOutivity/index.js

```diff
@@ -0,0 +1,156 @@
+import styled from "styled-components";
+import { useState } from "react";
+
+export default function EditOutivityForm({
+  onEditOutivity,
+  outivity,
+  setIsEdit,
+}) {
+  const [editedOutivity, setEditedOutivity] = useState({
+    id: outivity.id,
+    title: outivity.title,
+    area: outivity.area,
+    country: outivity.country,
+    image: outivity.image,
+    description: outivity.description,
+  });
+
+  const saveEdits = (event) => {
+    event.preventDefault();
+    onEditOutivity(editedOutivity);
+    setIsEdit(false);
+  };
+
+  return (
+    <StyledNewOutivitiesForm onSubmit={saveEdits}>
+      <h1>Edit Outivity</h1>
+      <StyledNewOutivitiesFormFields>
+        <StyledNewOutivitiesFormField>
+          <label htmlFor="outivityImage">Image</label>
+
+          <StyledNewOutivitiesFormInput
+            type="text"
+            name="outivityImage"
+            id="outivityImage"
+            value={editedOutivity.image}
```

Comment:
When you open the preview you will see, that your input fields cannot be edited. The reason for this is, that you give your input fields values. So they will have a value and cannot be edited anymore. 
I think what you want is to give it a ``defaultValue={outivity.image}``. The same for all other input-fields.

Reply:
Everything should work now :)